src/cljs/rems/actions/add_member.cljs

```diff
@@ -0,0 +1,142 @@
+(ns rems.actions.add-member
+  (:require [re-frame.core :as rf]
+            [reagent.core :as r]
+            [rems.actions.action :refer [action-button action-form-view button-wrapper]]
+            [rems.atoms :refer [textarea]]
+            [rems.autocomplete :as autocomplete]
+            [rems.status-modal :refer [status-modal]]
+            [rems.text :refer [text]]
+            [rems.util :refer [fetch post!]]))
+
+(defn fetch-potential-members
+  [[user on-success]]
+  (fetch (str "/api/applications/members")
+         {:handler on-success
+          :headers {"x-rems-user-id" (:eppn user)}}))
+
+(rf/reg-fx ::fetch-potential-members fetch-potential-members)
+
+(comment
+  (fetch-potential-members [{:eppn "developer"} prn]))
+
+(defn open-form
+  [{:keys [db]} _]
+  (merge {:db (assoc db
+                     ::potential-members #{}
+                     ::selected-members #{})}
+         {::fetch-potential-members [(get-in db [:identity :user])
+                                     #(rf/dispatch [::set-potential-members %])]}))
```

Comment:
why merge two literals?

Reply:
It was leftover code. I cleaned up the other actions too now.